nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/WorkManagerImpl.java

```diff
@@ -337,9 +359,28 @@ public int getApplicationStartedOrder() {
     @Override
     public void start(ComponentContext context) {
         super.start(context);
+        initDeadLetterQueueStream();
         init();
     }
 
+    protected void initDeadLetterQueueStream() {
+        StreamService service = Framework.getService(StreamService.class);
+        if (service == null) {
+            return;
+        }
+        try {
+            org.nuxeo.lib.stream.log.LogManager logManager = service.getLogManager(DEFAULT_LOG_MANAGER);
```

Comment:
You can use `var` here to avoid fully qualified name.

Reply:
interesting but I prefer to keep the legacy code in java8 to ease backport